src/client.py

```diff
@@ -1,10 +1,13 @@
 import json
 import os
+import time
 from json.decoder import JSONDecodeError
+from pathlib import Path
 
 import requests
```

Comment:
Cleaner to move this to its friends at the top 😸

Reply:
hmm, pretty sure i used our linters auto-structure for the imports, will check

Reply:
might have forgotten it on one of the files :D

Reply:
https://code.visualstudio.com/docs/python/editing#_sort-imports